test/e2e/standard/sanitychecker.go

```diff
@@ -37,9 +38,12 @@ type DeepTestInterface interface {
 }
 
 type SanityChecker struct {
-	Log    *logrus.Entry
-	cs     *internalapi.OpenShiftManagedCluster
-	Client *openshift.ClientSet
+	Log          *logrus.Entry
+	cs           *internalapi.OpenShiftManagedCluster
+	Client       *openshift.ClientSet
+	FakeRPClient *azure.Client
```

Comment:
The sanity checking mechanism only makes sense with kube/openshift clients and not azure clients. Please remove